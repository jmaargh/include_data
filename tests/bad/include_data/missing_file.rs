fn main() {}

// file doesn't exist
static BYTES_31: u128 = include_data::include_data!("../../test_data/non-existent");
