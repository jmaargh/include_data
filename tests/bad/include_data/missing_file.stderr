error: couldn't read $DIR/tests/bad/include_data/../../test_data/non-existent: No such file or directory (os error 2)
 --> tests/bad/include_data/missing_file.rs:4:25
  |
4 | static BYTES_31: u128 = include_data::include_data!("../../test_data/non-existent");
  |                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `::core::include_bytes` which comes from the expansion of the macro `include_data::include_data` (in Nightly builds, run with -Z macro-backtrace for more info)
